### DISCUSSION TITLE
Rancher Desktop 1.3.0

### DISCUSSION THREAD

**Discussion by jandubois** [rancher-sandbox/rancher-desktop#2126] [General]
This is the 1.3.0 release of Rancher Desktop, an open source desktop application to bring Kubernetes and container management to Mac, Windows, and Linux. The release contains some new features along with various bug fixes.

## Important Updates & Fixes

- On macOS and Linux, supporting utilities are no longer symlinked into `/usr/local/bin`, but into `~/.rd/bin` which can be added to the `PATH` automatically or manually in the shell profile files.

- On macOS Rancher Desktop can be configured to run without administrative rights, in which case it will not create the routable interfaces. It will also not be able to create a symlink at `/var/run/docker.sock`, but will setup an alternate docker context.

- `nerdctl` has been upgraded to 0.19.0.

- `helm` has been upgraded to 3.8.2.

## Experimental Features

- The `rdctl` CLI remains experimental and has added new subcommands:
  - `rdctl start --kubernetes-version 1.23.1` will start RD if it isn't running, or modify the configuration to match.
  - `rdctl shell` runs commands or opens an interactive shell inside the Rancher Desktop VM.

- An alternative DNS resolver for Windows has been implemented. It should support DNS lookup over VPN connections. It has to be enabled manually by editing an internal [configuration file](https://github.com/rancher-sandbox/rancher-desktop/issues/1899#issuecomment-1109128277).

- On macOS an alternative file sharing mechanism using `9p` instead of `reverse-sshfs` has been implemented. It is disabled by default. Talk to us on Slack if you want to help us testing it.

## Important Fixes

- The metrics performance issue when using the moby runtime has been fixed.
- Certificate handling on Windows has be fixed to deal with large numbers of certificates.
- Image scanning now displays **all** the reported vulnerabilities.

You can connect with the developers via:

- The [issue queue](https://github.com/rancher-sandbox/rancher-desktop/issues)
- [Rancher Users Slack](https://slack.rancher.io/) in the #rancher-desktop channel

## Changelog

The [full version changelog, from v1.2.1, can be found using GitHub compare](https://github.com/rancher-sandbox/rancher-desktop/compare/v1.2.1...v1.3.0) and the details of the release can be found in the [v1.3.0 milestone](https://github.com/rancher-sandbox/rancher-desktop/milestone/13?closed=1).

<hr /><em>This discussion was created from the release <a href='https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.3.0'>Rancher Desktop 1.3.0</a>.</em>

**Comment by Fran-Rg** [comment]
> On macOS an alternative file sharing mechanism using 9p instead of reverse-sshfs has been implemented. It is disabled by default. Talk to us on Slack if you want to help us testing it.

Any reason to not offer an option to enable it?

**Reply by jandubois** [reply]
> Any reason to not offer an option to enable it?

@Fran-Rg Because it was a very late addition, and it is not clear yet how well it works.

There are instructions on Slack at https://rancher-users.slack.com/archives/C0200L1N1MM/p1651620771539959?thread_ts=1651180737.038029&cid=C0200L1N1MM

**Reply by Fran-Rg** [reply]
Thanks, I'll look into it

**Comment by suuunly** [comment]
After updating to 1.3.0, I get this error when trying to get `docker ps`
`Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`

**Reply by jandubois** [reply]
> After updating to 1.3.0, I get this error when trying to get `docker ps`
> `Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?`

@suuunly If you have disabled administrative access ("sudo"), then you need to switch your docker context to `rancher-desktop` to make the socket discoverable by default:

```console
docker context use rancher-desktop
```

Alternatively you could set the `DOCKER_HOST` environment variable to point to the socket at `~/.rd/docker.sock`.
